cadence/contracts/LendingComptroller.cdc

```diff
@@ -41,45 +39,45 @@ pub contract LendingComptroller {
 
     /// Market
     ///
-    pub struct Market {
+    access(all) struct Market {
         /// Contains functions to query public market data
-        pub let poolPublicCap: Capability<&{LendingInterfaces.PoolPublic}>
+        access(all) let poolPublicCap: Capability<&{LendingInterfaces.PoolPublic}>
```

Comment:
Can this be `access(self)` or `access(contract)`. Usually it is safe to make fields like this private just to be safe

Reply:
Done